src/requirements/data/majors/fsad.ts

```diff
@@ -0,0 +1,233 @@
+import { Course, CollegeOrMajorRequirement } from '../../types';
+import { includesWithSubRequirements, ifCodeMatch, courseIsForeignLang } from '../checkers-common';
+
+const fashionDesignAdditionalRequirements: readonly string[] = [
+  'PBS',
+  'BIOLS-AG',
+  'BIONLS-AG',
+  'SBA',
+  'KCM',
+  'MQR',
+  'LA',
+  'CA',
+  'HA',
+];
+
+const fashionDesignNaturalScienceRequirements: readonly string[] = ['PBS', 'BIOLS-AG', 'BIONLS-AG'];
+
+const fashionDesignRequirements: readonly CollegeOrMajorRequirement[] = [
```

Comment:
see the comment i left in the main pr page for more details, but i found a few bugs relating to how AP courses count for these requirements, and there also might be additional bugs of a simliar nature that i did not catch.

Reply:
@benjamin-shen i think you saw this, but the ap classes have bugs 😭 it's not caused by this PR so I'm gonna add this bug to the notion to be tested before we push to prod.

as we talked off-line, maybe we potentially don't let ap classes automatically count for reqs, but limit it to user opt-in until we fix some of the bigger ap bugs??

the opt-in feature is gonna be added by andrew/will soon so at least there's a workaround (soon) for now

Reply:
we're temp not automatically applying ap credits (pr: https://github.com/cornell-dti/course-plan/pull/676#pullrequestreview-928913970), so this problem should be temp fixed rn!